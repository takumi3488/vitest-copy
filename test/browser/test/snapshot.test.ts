import { expect, test } from 'vitest'

test('inline snapshot', () => {
  expect(1).toMatchInlineSnapshot('1')
})

test('snapshot', () => {
  expect(1).toMatchSnapshot()
})

test('file snapshot', async () => {
  await expect('my snapshot content')
    .toMatchFileSnapshot('./__snapshots__/custom/my_snapshot')
})
