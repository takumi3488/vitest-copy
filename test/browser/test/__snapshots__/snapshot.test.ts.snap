// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`snapshot 1`] = `1`;
